README.md

```diff
@@ -1,2 +1,13 @@
 # open-source-teams
 Place to document everything about our AnitaB.org Open Source teams
+
+Hey there!
+
+If you are here, it probably means you are either looking for a place to start contributing, or some direction regarding your current task. This is the place you will find our community's documentation common to all projects. Below is a list of documents currently in this repository. If none of these documentation helps, feel free to reach out to us either by asking your question on Zulip or creating an issue requesting the information you need.
+
+## The documents in this repository:
+- [Code of Conduct](https://github.com/Frenzyritz13/documentation/blob/master/Contributing/CODE_OF_CONDUCT.md)
+  - This document covers all the points regarding what is expected of you as a member of this community, it lists down appropriate and unappropriate behaviour.
+- [Reporting Guidelines](https://github.com/Frenzyritz13/documentation/blob/master/Contributing/REPORTING_GUIDELINES.md)
```

Comment:
```suggestion
- [Reporting Guidelines](/Contributing/REPORTING_GUIDELINES.md)
```
this suggestion is to use relative path, otherwise, someone in this repository when clicking these links would be redirected to your fork